### ISSUE TITLE
Need some advices on fine tuning settings

### ISSUE THREAD

**Issue by robbine** [open-mmlab/mmclassification#727]
After reading this doc https://mmclassification.readthedocs.io/en/latest/tutorials/schedule.html , 
I find that there are so many different options , such as warm up, learning rate decay,  and etc.
Would you give some specific advices on fine tuning ? For example batch size number, scheduler choice, optimzier choice for beginners like me . Thanks a lot !

**Comment by Ezra-Yu** [comment]
You can refer to this [ebook](https://d2l.ai/index.html), especially the chapter 11 [Optimization Algorithms](https://d2l.ai/chapter_optimization/index.html#) and chapter 13 [Computer Vision](https://d2l.ai/chapter_computer-vision/index.html) for some setting, such as `schedule`, `optimizer`, `data augments`. And other settings may need experience.

**Comment by Ezra-Yu** [comment]
Normally, we use 'sgd' to train the CNN base network and 'adam to train the Vit base network.
It is better to train a network using warming up. And the 'CosineAnnealing' is the most used lr-decay scheduler.
and the batch-size, max_epoch, lr changes according your dataset.

**Comment by robbine** [comment]
Beginners like me benefit tremendously from these guidelines. Thank you so much !
